src/flb_log.c

```diff
@@ -571,13 +566,13 @@ int flb_log_construct(struct log_message *msg, int *ret_len,
     int ret;
     int len;
     int total;
-    time_t now;
```

Comment:
If you want to change `flb_log_construct` to be `static`. I'm only mentioning it because you had originally moved the type definition and added the prototype which I assume was done for correctness sake.